server/repository/migrations/postgres/2022-01-27T08-00_create_changelog_table/up.sql

```diff
@@ -18,47 +18,52 @@ CREATE TYPE row_action_type AS ENUM (
 );
 
 CREATE TABLE changelog (
-    id BIGSERIAL,
+    cursor BIGSERIAL,
     -- the table name where the change happend
     table_name changelog_table_name NOT NULL,
     -- row id of the modified row
-    row_id TEXT NOT NULL,
+    record_id TEXT NOT NULL,
     -- Sqlite only fires INSERT when doing an upsert (it does a delete + insert) for this reason
```

Comment:
This sqlite comment is confusing to me. Is it saying that we shouldn't ever do an upsert or that we always should do upsert?

Reply:
I actually removed the comment, I don't think it serves much purpose, it's quite clear in trigger logic: e45915ba2bea82534d104af82b92bf48cffd11c8